app/pages/ppr-marketing.vue

```diff
@@ -73,6 +75,13 @@ const { data: pprSections } = await useAsyncData(`ppr-sections-${locale.value}`,
   return queryContent().where({ _locale: locale.value, _path: { $contains: 'ppr/sections' } }).find()
 })
 
+// srr safe media query
+watchDebounced(width, () => {
```

Comment:
interesting, for hydration issues?

Reply:
Yes, initial render the ref is false on server and true on client so it causes a mismatch. This way the value is the same for server/client and still updates when the screen width changes, used debounced watcher to update once the media query ref changes and to not fire multiple events if the screen changes width after mount.